apps/experiments/views/experiment.py

```diff
@@ -1170,3 +1170,15 @@ def set_default_experiment(request, team_slug: str, pk: int):
     experiment.save()
 
     return redirect("experiments:versions-list", team_slug=request.team.slug)
+
+
+@require_POST
+@transaction.atomic
+@login_and_team_required
+def archive_experiment(request, team_slug: str, pk: int):
+    experiment = get_object_or_404(Experiment, id=pk)
+    experiment.is_archived = True
```

Comment:
what happens if it's also the default?

Reply:
modify such that the user much change the default prior to archiving. I thought about making a different default for them, but feels risky not letting the user decide so opted for this approach